source/extensions/filters/http/transformation/inja_transformer.h

```diff
@@ -47,6 +47,9 @@ class TransformerInstance {
   void set_element_notation(inja::ElementNotation notation) {
       env_.set_element_notation(notation);
   };
+  void set_escape_strings(bool escape_strings) {
```

Comment:
I think a comment here could be useful. I'm thinking about something along the lines of:
`// Sets the config for rendering strings raw or escaped`

Reply:
good call; added! 46f2a5f2597f208270d96a91568d61c2c17550a0